lib/brains/tests/test_token.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+from solitude.base import APITest
+
+from nose.tools import eq_
+from django.core.urlresolvers import reverse
+
+
+class TestToken(APITest):
+
+    def setUp(self):
+        self.url = reverse('braintree:token.generate')
+
+    def test_token(self):
```

Comment:
I know it's a bit paranoid but it might be nice to add a test from an anonymous client that asserts a 403